javascripts/stageprompt.js

```diff
@@ -62,7 +62,11 @@
   }())
 
   GOVUK.performance.sendGoogleAnalyticsEvent = function (category, event, label) {
-    global._gaq.push(['_trackEvent', category, event, label, undefined, true])
+    if (global.ga && typeof global.ga === 'function') {
+      global.ga('send', 'event', category, event, label)
```

Comment:
The sixth element in the `trackEvent` array means means that `opt_noninteraction` is `true`, which [means that the 'event hit will not be used in bounce-rate calculation'](https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide#setting-up-event-tracking).

Assuming we'd want the same behaviour with GA Universal, I think we'd want to make this:

```
global.ga('send', 'event', category, event, label, {
  nonInteraction: true
})
```

as per https://developers.google.com/analytics/devguides/collection/analyticsjs/events#non_interaction_events?

I'm not a performance analyst however… there may be other better ways of achieving the same thing without hardcoding it.

Reply:
It seems sensible to mirror them.

@rpowis would be good to get your thoughts on this.

Reply:
yeah good spot @36degrees. i think this got overlooked for whatever reason in our forked version. 

so as things stand, any page using universal tracking with an event on it has been adding to the bounce rate if that event hasn't been triggered? 😬 

similar to the `_gaq` call, we probably don't want to add to the bounce rate in the majority of cases so defaulting to `{ nonInteraction: true }` would make sense. but i think having the ability to opt-in would be good.

that said, i don't think any services here are getting that granular with their analytics so i'm not sure it would have to be done as part of this PR.